API V3: Don't allow leaking teams through expandable fields (#11471)
readthedocs/readthedocs.org: docs/user/api/v3.rst

      :Query Parameters:

>          * **expand** (*string*) -- Allows to add/expand some extra fields in the response.

Reviewer: ```suggestion
         * **expand** (*string*) -- Add additional fields in the response.
```

      :Query Parameters:

>          * **expand** (*string*) -- Allows to add/expand some extra fields in the response.

Reviewer: ```suggestion
         * **expand** (*string*) -- Add additional fields in the response.
```

    .. tabs::

        .. code-tab:: bash

            $ curl -H "Authorization: Token <token>" https://readthedocs.com/api/v3/organizations/pypa/teams/

        .. code-tab:: python

            import requests
            URL = 'https://readthedocs.com/api/v3/organizations/pypa/teams/'
            TOKEN = '<token>'
            HEADERS = {'Authorization': f'token {TOKEN}'}
            response = requests.get(URL, headers=HEADERS)
            print(response.json())

    **Example response**:

    .. sourcecode:: json

       {
          "count": 2,
          "next": null,
          "previous": null,
          "results": [
             {
                   "access": "admin",
                   "created": "2024-07-01T21:19:55.573819Z",
                   "modified": "2024-07-01T21:19:55.573837Z",
                   "name": "Admins",
                   "slug": "admins"
             },
             {
                   "access": "readonly",
                   "created": "2024-07-01T21:19:55.593815Z",
                   "modified": "2024-07-01T21:19:55.593829Z",
                   "name": "Read Only",
                   "slug": "read-only"
             }
          ]
       }

>     :query string expand: Allows to add/expand some extra fields in the response.

Reviewer: ```suggestion
    :query string expand: Add additional fields in the response.
```
